apps/studio/components/ui/AIAssistantPanel/AIAssistant.tsx

```diff
@@ -52,6 +53,7 @@ const MemoizedMessage = memo(
       resultId?: string
       results: any[]
     }) => void
+    onEdit?: (id: string) => void
   }) => {
     return (
       <Message
```

Comment:
Could we set opacity to 50 for any messages that are after the message being edited, and if those messages are clicked, it cancels the edit

Reply:
also added a pulse animation to the message being edited
[`dcc90f2` (#37603)](https://github.com/supabase/supabase/pull/37603/commits/dcc90f28da096a28f1f8b8dbe591102375b71b13)

![demo](https://github.com/user-attachments/assets/3f2996b7-8b4f-4ba7-a6e1-f02df39a85fb)